PublicGitArchive/doc/dataset_analysis.md

```diff
@@ -0,0 +1,66 @@
+# Description of the current dataset
+
+## 1. GHTorrent: selecting the list of URLs
+
+From GHTorrent’s MySQL dump, dated from 01/01/2018.
+By querying repos with at least 50 stargazers
+
+Output: list of 187,352 URLs  
+
+## 2. Collection with Borges
+
+Output of Borges:  
+
+182,014 **repositories**  
+5,338 are **indefinitely pending** for several reasons :  
+* 3,156 became inaccessible by the Git clone time because :
+    * they became private
+    * the did not exist anymore
+* 82 removed for legal reasons
+* borges was not able to process the rest  
+
+## 4. Latest CSV file : `dataset_stats.ipynb`
+
+Number of **rows/urls** : 182,014  
+Number of **siva files** : 248,043  
+Number of **forks** : 59,246  
+Total size : 2.96 TB
+
+Number of distinct **languages** : 455  
+In Linguist/enry, we count 340 languages of type: programming and 469 overall. Thus, it includes also non programming languages.
+
+Total number of **files** : 54.5 million  
+Total number of **lines** : 15,941 million  
+Total number of **bytes** : 710 GB  
+
+## 5. Dataset Analysis given the latest CSV file : `size_analysis_siva_files.ipynb`
+
+* Repositories that count the most siva files, [related issue](https://github.com/src-d/borges/issues/222)
+
+|   | Number of siva files | URL | Total size | min/max size | Average size |  
+|---|----------------------|-----|------------|--------------|--------------|
+| 1. | 6,613 | github.com/google/angle |  ~     |  ~  |  ~  |
```

Comment:
Why are these sizes empty?

Reply:
@warenlg Indeed

Reply:
True, I'm on it

Reply:
It was

Reply:
Yes I fixed it in another PR https://github.com/src-d/datasets/pull/38